adapter/config/default_config.go

```diff
@@ -139,6 +139,24 @@ var defaultConfig = &Config{
 					},
 				},
 			},
+			CircuitBreakers: []upstreamCircuitBreaker{
+				{
+					Organizations:      "*",
+					CircuitBreakerName: "BasicCircuitBreaker",
+					MaxConnections:     3,
+					MaxRequests:        3,
+					MaxPendingRequests: 1,
+					MaxRetries:         3,
+				},
+				{
+					Organizations:      "e0682456-2ba6-4c5f-8f36-3c5b6dc46913,4b9afefb-4bcc-4e63-85d3-ddd593841012,d3a7dfea-fb10-4371-b21d-85d1bc28667b",
```

Comment:
let's keep empty here and add these from cp-env-overlay I guess.
It would be cleaner IMO.